Tests/AblyChatTests/SubscriptionTests.swift

```diff
@@ -21,4 +21,36 @@ struct SubscriptionTests {
 
         #expect(await emittedElements == ["First", "Second"])
     }
+
+    @Test
+    func addTerminationHandler_terminationHandlerCalledWhenSubscriptionDiscarded() async throws {
+        let onTerminationCalled = AsyncStream<Void>.makeStream()
+
+        ({
```

Comment:
I'm not familiar with this, these are braces of what?

Reply:
It's just a function expression which I then invoke. (I used a function to give more confidence about by which moment `subscription` would be deallocated.)